doc/whats-new.rst

```diff
@@ -32,6 +32,10 @@ New Features
   By `Miguel Jimenez <https://github.com/Mikejmnez>`_ and `Wei Ji Leong <https://github.com/weiji14>`_.
 - Unary & binary operations follow the ``keep_attrs`` flag (:issue:`3490`, :issue:`4065`, :issue:`3433`, :issue:`3595`, :pull:`4195`).
   By `Deepak Cherian <https://github.com/dcherian>`_.
+- :py:attr:`DataArray.rolling` and :py:attr:`Dataset.rolling` now also respect the ``keep_attrs``
+  keyword for their ``DataArray``s (previously only the global attrs were retained). ``keep_attrs``
```

Comment:
```suggestion
  keyword for wrapped variables (previously only the global attrs were retained). ``keep_attrs``
```
wording aside (it seems strange to me but I'm not a native speaker, and I think my suggestion is not really accurate either), characters after the double backticks are not supported by `rst`. The workaround I normally use is
```rst
``<type>`` objects
```